README.md

```diff
@@ -1,8 +1,64 @@
+
+## Skills
+
+**Proficient:** Java, Python, Linux/Bash, RDBMS/SQL, Agile/Scrum, Git/GitFlow/GitHub
+
+**Competent:** Docker, MongoDB, Cloud platforms (GCP,AWS), REST APIs, Numpy/Pandas
+
+**Familiar:** Spark, Scala, Hadoop, HTTP Requests, Apache Cassandra
+
+## Jarvis Projects
+
+Project source code: [https://github.com/jarviscanada/jarvis_data_eng_NasrSyed](https://github.com/jarviscanada/jarvis_data_eng_NasrSyed)
+
+
+**Linux Cluster Monitoring Agent** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_NasrSyed/tree/master/linux_sql)]: Architected a routine monitoring agent script which tracked a cluster of nodes/servers running on CentOS 7 Linux distribution. Bash scripts retrieving host hardware, CPU and Memory usage were deployed, and provisioned via Docker. Results were periodically stored into a PostgreSQL database via crontab job.
+
+**Core Java Apps** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_NasrSyed/tree/master/core_java)]:
+      
+  - Grep App: Implemented a Java application that takes in parameters for a regex pattern and returns matching lines containing this pattern from all text files in the target directory to an output text file. 
+  - JDBC App: Architected a Java application that connects to a PSQL database containerized in Docker to perform basic CRUD operations between an object and respective database.
+  - Twitter App: Curabitur laoreet tristique leo, eget suscipit nisi. Sed in sodales ex. Maecenas vitae tincidunt dui, et eleifend quam.
+
+**Springboot App** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_NasrSyed/tree/master/springboot)]: Not Started
+
+**Python Data Analytics** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_NasrSyed/tree/master/python_data_anlytics)]: Not Started
+
+**Hadoop** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_NasrSyed/tree/master/hadoop)]: Not Started
+
+**Spark** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_NasrSyed/tree/master/spark)]: Not Started
+
+**Cloud/DevOps** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_NasrSyed/tree/master/cloud_devops)]: Not Started
+
+
+## Highlighted Projects
+**ETL Project for Market Exchange Data**: Extracted exchange rate data from all countries from CSV, JSON and XML files into a dataframe, transformed currency data, and loaded data into target CSV file. Formulated logging function to track/provide timestamps and logging details of ETL process. (Python, Pandas)
```

Comment:
**ETL Project for Market Exchange Data**: Extracted exchange rate data from all countries from CSV, JSON and XML files into a data frame, transformed currency data and loaded data into a target CSV file. Formulated logging function to track/provide timestamps and logging details of the ETL process. (Python, Pandas)
		
fixed grammar